jenkins/automation/kubernetes/upgrade-cortx-cluster.groovy

```diff
@@ -53,7 +53,7 @@ pipeline {
                 script { build_stage = env.STAGE_NAME }
                 sh label: 'fetch cluster status', script: '''
                     pushd solutions/kubernetes/
-                        echo $hosts | tr ' ' '\n' | head -n 1 > hosts
+                        echo $hosts | tr ' ' '\n' > hosts
```

Comment:
We need only primary node details for upgrade right so can we continue using `head -n 1`

Reply:
@shailesh-vaidya, All nodes are needed as provisioner suggested us to download docker images on all nodes so upgrade won't go in timeout stage as sometimes it takes time to download images on worker nodes and reaches time out so upgrade fails.